book/src/docker.md

```diff
@@ -61,36 +61,35 @@ ${version}${arch}${stability}${modernity}${features}
 
 The `version` is:
 
-* `vX.Y.Z` for a tagged Lighthouse release, e.g. `v2.1.1`
-* `latest` for the `stable` branch (latest release) or `unstable` branch
+-   `vX.Y.Z` for a tagged Lighthouse release, e.g. `v2.1.1`
+-   `latest` for the `stable` branch (latest release) or `unstable` branch
 
 The `arch` is:
 
-* `-amd64` for x86_64, e.g. Intel, AMD
-* `-arm64` for aarch64, e.g. Raspberry Pi 4
-* empty for a multi-arch image (works on either `amd64` or `arm64` platforms)
+-   `-amd64` for x86_64, e.g. Intel, AMD
+-   `-arm64` for aarch64, e.g. Raspberry Pi 4
+-   empty for a multi-arch image (works on either `amd64` or `arm64` platforms)
 
 The `stability` is:
 
-* `-unstable` for the `unstable` branch
-* empty for a tagged release or the `stable` branch
+-   `-unstable` for the `unstable` branch
```

Comment:
I don't see how our editorconfig is causing these changes. As far as I can tell we haven't said "prefer `*` to `-`". Similarly with indentation, why are bodies of the bullet points getting indented?

Do you mind letting us know which editor you're using and which markdown linter it is using? I agree with the idea of linting the book markdown so it's consistent, but would prefer to do that in a separate PR and keep the changes in this PR to a minimum (you could `git cherry-pick -p` to select just the relevant lines).

Reply:
Seems to be due to vscode doing format on save with prettier whenever a configuration file is present (by default prettier will look for either a ``.prettierrc`` file, or otherwise an ``.editorconfig`` file).

Would have been avoided if I didn't have a global installation of prettier / auto format on save.

Prettier uses ``-`` instead of ``*`` for unordered lists (introduced in https://github.com/prettier/prettier/pull/4440) and the indentation seems to be an ongoing issue (https://github.com/prettier/prettier/issues/5019).

I've rebased onto the most recent merged unstable commit and applied only the relevant changes to this PR. Definitely makes the diff a lot cleaner!